hexa/pipelines/views.py

```diff
@@ -102,7 +102,7 @@ def run_pipeline(
         HttpResponse: Returns a dict with the `run_id` key containing the ID of the created run.
     """
     try:
-        pipeline = Pipeline.objects.get(id=id)
+        pipeline = Pipeline.objects.get(webhook_token=id)
```

Comment:
The type of the `id` argument is not the right one. It can be a string as well.

Reply:
I guess there is the same problem with the urls.py.

Reply:
This is not what I had in ming. You have to support the two solutions to run pipelines, with an uuid and a generated signed token. We have to give time to people to switch to the new webhook token. The implementation in this file would be to try to validate the token and if it's not correct, try to get the pipeline with the id

Reply:
Okay